src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -181,11 +182,19 @@ export class NfsFormComponent extends CdForm implements OnInit {
   }
 
   volumeChangeHandler() {
-    this.pathChangeHandler();
-    const fs_name = this.nfsForm.getValue('fsal').fs_name;
+   const fs_name = this.nfsForm.getValue('fsal').fs_name;
+    const subvolgrp = this.nfsForm.getValue('subvolume_group');
     this.getSubVolGrp(fs_name);
-  }
+    this.getSubVol();
 
+    if(subvolgrp == this.defaultSubVolGroup){
+      this.updatePath('/volumes/' + this.defaultSubVolGroup);
+      this.setUpVolumeValidation();
```

Comment:
This whole logic can go in a separate function.
And reuse here and in `setSubVolGrpPath`